### ISSUE TITLE
bug: text-field "0000" hint truncated to "0"

### ISSUE THREAD

**Issue by vbresults** [NativeScript/NativeScript#3983]
```xml
<text-field hint="0000" />
```

This displays as 0 instead of 0000 on iOS.  Might be for `text-view` as well, haven't tested that.

**Comment by tsonevn** [comment]
Hi @vbresults,
You should use the `text` property for the `TextField` instead of `value` . 
For example:
```
<TextField hint="" text="0000" />
```
Then the text should be displayed properly.

**Comment by vbresults** [comment]
@tsonevn My apologies, I meant to post this about the hint not the value.

**Comment by tsonevn** [comment]
Hi @vbresults,
Indeed this is a valid issue with the `TextField` `hint` property while using modules `2.5`,
On that point, I have tested this scenario with our [@rc modules ](https://www.nativescript.org/blog/nativescript-3.0-release-candidate-available-today) and this problem seems to be fixed.

**Comment by SvetoslavTsenov** [comment]
The fix is already available in tns-core-modules@next

**Comment by dtopuzov** [comment]
Fix will be released in 3.0.1 later this week.
